### ISSUE TITLE
JetEngine - Count user posts

### ISSUE THREAD

**Issue by joseehilton147** [CrocoBlock/suggestions#2241]
hello, its possible to add a function to count users posts, another option on this jet engine user info menu like total users posts

![1111](https://user-images.githubusercontent.com/53986027/100828547-1d5a3580-343e-11eb-81d5-ac037cccc034.png)

**Comment by DmitryBychenko** [comment]
Would be nice to have feature to display number of *items*,
for example, 
number of pages, posts, cpt's...
number of users..
categories...

Useful to display it on home pages:
this amount of users
this amount of posts or cpt's (books, movies, ... content)
this many bla bla.

feature to round numbers

**Comment by joseehilton147** [comment]
Yes, would be nice if we could have this... for now im using this, its a little code that can be used as shortcode, so i put it anywhere i want, its for all posts (my need at the moment)

Insert this into functions.php: 
```
function total_user_post_shortcode() {
    $current_user = wp_get_current_user();
    if ( 0 != $current_user->ID ) {
    echo $user_post_count = count_user_posts( $current_user->ID );
   }
}
```
add_shortcode('total-user-posts', 'total_user_post_shortcode');`

and the shortcode is: `[total-user-posts]`

Only display for logged user, ive done a button who display a sidebar popup and my client need to set that on footer of that sidebar...
 
![2](https://user-images.githubusercontent.com/53986027/101313859-0187d280-3836-11eb-874c-4a4311a9c7a2.png)

**Comment by kervis** [comment]
code for show for public users?

**Comment by DaveKani** [comment]
How would I change the PHP to count certain CCT's the user created?

**Comment by aalexandr21** [comment]
Hi 
You can use the Query Builder to count posts for specific users and then [use the Query Results Count Dynamic Tag](https://crocoblock.com/knowledge-base/jetengine/query-results-count-dynamic-tag-for-wordpress/) 

<img width="1263" height="595" alt="Image" src="https://github.com/user-attachments/assets/0405f8a8-9ebe-4c0d-9afd-e8e55b79880e" />
